styles/styles.go

```diff
@@ -59,20 +61,41 @@ func SelectTheme(theme string) glamour.TermRendererOption {
 	case "notty":
 		return glamour.WithStyles(glamour.NoTTYStyleConfig)
 	default:
-		bytes, err := os.ReadFile(theme)
+		var themeReader io.Reader
+		var err error
+		if strings.HasPrefix(theme, "http://") || strings.HasPrefix(theme, "https://") {
```

Comment:
I think this is good enough

```suggestion
		if strings.HasPrefix(theme, "http") {
```

Reply:
What if it's something like `httpie.xxx`?